recipe/meta.yaml

```diff
@@ -9,16 +9,16 @@ source:
   sha256: d4142d4d56d46f449d6f00536a73625a871cba040f0bc1a2e305a04578f07d1e
 
 build:
-  number: 1002
-  skip: True  # [not win]
```

Comment:
Would this be broken on non-windows? I don't know this package well enough to be honest.

Reply:
It is pure python https://github.com/Drekin/win-unicode-console and at least the import works on Linux as well. But you are right there are packages like https://github.com/bontchev/pcodedmp/blob/master/pcodedmp/pcodedmp.py#L12 which just check if the package exists rather than checking the operation system.